src/mono/wasm/runtime/loader/logging.ts

```diff
@@ -1,6 +1,30 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+/* eslint-disable no-console */
+import { loaderHelpers } from "./globals";
+
+export function mono_log_debug(msg: string, ...data: any) {
+    if (loaderHelpers.diagnosticTracing) {
+        console.debug("MONO_WASM: " + msg, ...data);
+    }
+}
+
+export function mono_log_info(msg: string, ...data: any) {
+    console.info("MONO_WASM: " + msg, ...data);
+}
+
+export function mono_log_info_no_prefix(msg: string, ...data: any) {
+    console.info("MONO_WASM: " + msg, ...data);
```

Comment:
`all of them prefix with MONO_WASM:  except mono_log_info_no_prefix which doesn't`
here it's with the prefix